ledger.js

```diff
@@ -0,0 +1,152 @@
+const Eos = require("eosjs");
```

Comment:
ledger.js should not be committed to the ledger-ui repository. Instead, it should be included in `package.json` as `volentix-ledger`. If we don't do this, changes made to the ledger-js won't be picked up by ledger-ui